internal/errors/errors.go

```diff
@@ -0,0 +1,78 @@
+package errors
```

Comment:
@jon-whit this code looks like a copy of https://www.reddit.com/r/golang/comments/ue73jz/comment/i6ob6os/

Can we use the library instead and give attribution? https://github.com/natefinch/wrap